code/modules/mob/living/carbon/alien/damage_procs.dm

```diff
@@ -5,9 +5,4 @@
 /mob/living/carbon/alien/adjustToxLoss(amount, updating_health = TRUE, forced = FALSE) //alien immune to tox damage
 	return FALSE
 
-//aliens are immune to stamina damage.
-/mob/living/carbon/alien/adjustStaminaLoss(amount, updating_health = 1)
```

Comment:
are you sure you want xenos to be stunned by disablers?

Reply:
Ye